src/sage/combinat/k_regular_sequence.py

```diff
@@ -1204,14 +1223,37 @@ def guess(self, f, n_max=100, max_dimension=10, sequence=None):
         zero = domain(0)
         one = domain(1)
 
+        # A `line` will be a triple `(t, r, s)` corresponding to an entry
+        # `k**t * m + r` belonging to `M_s`
+        # TODO: what is `M_s`?
+
+        # The elements of `lines` will be correspond to the current components
```

Comment:
```suggestion
        # The elements of `lines` will correspond to the current components
```

Reply:
Copied to https://github.com/sagemath/sage/pull/35682/files#r1208409597